build.gradle

```diff
@@ -7,64 +7,57 @@ Base directory: $projectDir
 Running script ${relativePath(buildFile)}
 """
 
-apply plugin: 'idea'
-apply plugin: 'eclipse'
 apply plugin: 'groovy'
 apply plugin: 'maven'
-apply plugin: 'git'
```

Comment:
I am not using any IDE, but according to the [documentation](https://docs.gradle.org/current/userguide/idea_plugin.html) the plugins generate tasks that will generate IDE-specific configuration files, so I assume they are useful for somebody. At least they were added (which was prior to me working on this plugin) for a reason.